### DISCUSSION TITLE
Video conversion

### DISCUSSION THREAD

**Discussion by guerinoni** [pion/webrtc#2453] [General]
Hi ✋🏻 I'm working on RTMP server that takes stream and push to webrtc using `TrackLocalStaticRTP`, what I can see is that on my call I'm able to send audio correctly and it is produced correctly but video is not started like is not published. I see a debug msg saying `video is not VP8` and it's true because I'm sending h264 from ffmpeg (video example of bunny in the garden :) )... 
My question is, are there any pkg to convert the video data from H264 into VP8?

**Comment by onthegit** [comment]
chrome supports h264 for webrtc. firefox no

**Comment by onthegit** [comment]
Show example code
